tests/unit/automatic_checks/test_automatic_sanity_checks.py

```diff
@@ -91,3 +92,60 @@ def test_polarity_switched(self):
         assert report.something_wrong == True
         assert report.ean == ean
         assert report.reasons == expected_reasons
+
+    def test_quality_passable(self):
+        # Arrange
+        ean = 123
+        predicted = pd.DataFrame(
+            data={"predicted": range(1000), "realised": range(1000)}
+        )
```

Comment:
To properly mock a fake realized/predicted dataframe, consider using a date_range as index, i.e.:
```suggestion
        predicted = pd.DataFrame(
            data={"predicted": range(1000), "realised": range(1000)},
            index=pd.date_range(start="2022-01-01", periods=1000, freq="15T"),
        )
```

Reply:
The same is true for the other unit tests.